src/main/java/com/treasure_data/td_import/Constants.java

```diff
@@ -95,6 +95,7 @@
             "    --column-types TYPE,TYPE,...     column types [string, int, long, double]\n" +
             "    --column-type NAME:TYPE          column type [string, int, long, double]. A pair of column name and type can be specified like 'age:int'\n" +
             "    -S, --all-string                 disable automatic type conversion\n" +
+            "    --empty-as-null                  the empty string values are interpreted as null values if columns arenumerical types." +
```

Comment:
"empty-as-null" sounds it works for String type as well (but not true I assume.) How about --empty-numeric-as-null?